trade-finance-manager-ui/templates/utilisation-reports/_macros/fee-record-details-table.njk

```diff
@@ -14,7 +17,7 @@
   <form method="post">
     <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
     <table class="govuk-table" data-module="moj-sortable-table" data-cy="fee-record-details-table">
-      <caption class="govuk-table__caption govuk-table__caption--m">{{ captionText }}</caption>
+      <caption class="govuk-table__caption govuk-table__caption--m" id="addedReportedFeesDetails" tabindex="-1">{{ captionText }}</caption>
```

Comment:
I think it is typical to use lower case hyphen separated words for html ids e.g. 'added-reported-fees-details', although here I think we can even make the name more specific whilst we are at it: 'added-reported-fees-details-header' to make it extra clear?

Reply:
Ah yep, I think the existing cases I saw were linking to govuk input components (e.g. bankDecisionDate) - I'll modify to this kebab casing thanks

Reply:
Resolved in ca77aa96a3460aa75aa51f101eb537bc29e40bbf